com.regnosys.rosetta/model/Rosetta.xcore

```diff
@@ -307,30 +312,59 @@ class RosettaSegment extends RosettaRootElement, RosettaNamed {
  * Mappings
  */
 
+interface RosettaMapTest {}
+
+interface RosettaMapTestExpression extends RosettaMapTest {}
```

Comment:
I separated the rosetta map test expression syntax from the rosetta expression syntax.